test/organizationMemberships.matchers.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const { _, expect, definePattern } = require('./matchers');
+
+definePattern(
+  'anOrganizationMembership',
+  {
+    uid: _.isUid,
+    createdAt: _.isISODateString,
+    organizationApikey: _.isOrganizationApikey,
+    memberUserDid: _.isDID,
+    admin: _.isUndefinedOr(_.isBoolean),
+    curator: _.isUndefinedOr(_.isBoolean),
+    updatedAt: _.isUndefinedOr(_.isISODateString),
+  }
+);
+
+definePattern(
+  'anOrganizationAdminMembership',
+  target => {
+    expect(target).to.be.anOrganizationMembership();
+    expect(target.admin).to.be.true;
```

Comment:
```suggestion
    expect(target).to.matchPattern({ admin: true, '...': 1 });
```

yields a nicer error message

Reply:
cool, done